frontend/init-pro-icons.sh

```diff
@@ -1,7 +1,28 @@
-#!/bin/sh -eu
+#!/bin/bash
 
-# SPDX-FileCopyrightText: 2017-2020 City of Espoo
+# SPDX-FileCopyrightText: 2017-2023 City of Espoo
 #
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
-aws --profile voltti-local s3 sync s3://evaka-deployment-local/frontend/vendor/fortawesome/ ./vendor/fortawesome/
+set -euo pipefail
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+VERSION="6.4.0"
+
+unpack() {
+  name=${1}
+  npm pack "@fortawesome/${name}@${VERSION}"
+  mkdir -p "node_modules/@fortawesome/${name}"
+  tar xz --strip-components 1 -C "node_modules/@fortawesome/${name}" -f "fortawesome-${name}-${VERSION}.tgz"
+  rm "fortawesome-${name}-${VERSION}.tgz"
```

Comment:
`npm pack` + `tar x` combo seems unnecessary, perhaps just `npm install --no-save ${name}@${VERSION}` would be enough?

Reply:
Tested `npm install`. The `package.json` contains `link:` -dependencies that are not compatible with node 18. Also one common package is installed with `npm install`. Using pack + tar for now.